libs/ledger-live-common/package.json

```diff
@@ -70,7 +70,7 @@
     "updateAppSupportsQuitApp": "node scripts/updateAppSupportsQuitApp.js",
     "prettier": "prettier --write 'src/**/*.?s' 'cli/src/**/*.?s'",
     "lint": "eslint src",
-    "lint:fix": "pnpm lint --fix",
+    "lint:fix": "pnpm lint --fix --quiet",
```

Comment:
Just curious, is this change required here?

Reply:
My bad, I just sometimes need to only have error, i'll fix